Requesting changes

Pgarza-dev opened the pull request.

sulenchy reviewed (changes requested):
@Pgarza-dev You have a good design and it is perfectly responsive. However, there is one expectation yet to be met. You need show you can extend bootstrap classes or variables. 

https://content.noroff.dev/css-frameworks/bootstrap-with-sass.html

Pgarza-dev commented:
Thanks for the input Abi! I've added my custom variables in my (_variables.scss) file! Should I resubmit on moodle?

sulenchy commented on src/scss/_variables.scss:
```diff
@@ -0,0 +1,1746 @@
+// Variables
+//
```
I think there is a misunderstanding here. You should not be lifting all the Bootstrap variables. You should just extend the few ones you need in your project. 

For instance, you might want to change the shades of blue that come with bootstrap. So all you have to do is to add this here 
`$blue:    #5d95e8;`

And that is all.

I think you need to go through this again https://content.noroff.dev/css-frameworks/bootstrap-with-sass.html#bootstrap-variables

sulenchy replied:
Can you explain why you added all these bootstrap variables to this file? Did you go through this lesson https://content.noroff.dev/css-frameworks/bootstrap-with-sass.html#bootstrap-variables?

sulenchy commented on src/scss/index.scss:
```diff
@@ -1,7 +1,8 @@
-@import "../../node_modules/bootstrap/scss/bootstrap";
 
+@import "../../node_modules/bootstrap/scss/bootstrap";
+@import "_variables.scss";
```
This does not look good. This should come before line 4. And also, it should be
`@import "variables.scss";`

sulenchy replied:
How about this? See this https://content.noroff.dev/css-frameworks/bootstrap-with-sass.html for reference

sulenchy reviewed (changes requested):
@Pgarza-dev Here is another chance to pass this CA. I just added new comment for you. Failure to implement this before the end of today will amount to missing an important requirement of the CA and that is an auto failure.

Pgarza-dev commented:
Hi Abi. I have made the changes you suggested even though I had already made this requirement previously. I originally had changed the "Primary" scss variable to  $Clear_Cyan: #26BFBF. I have now changed a second "$blue:    #5d95e8".  Maybe you missed it the first time because I wanted to keep the naming consistent with the original Bootstrap file "primary" name. I also changed, my variable.scss file I name. But the original naming convention Oliver had told us was okay. I have been busy with my son's baptism all day today so Im sending this message very late in the even. I hope this will get to you in time!!

Pgarza-dev commented:
Hi Abi. I have made the changes you suggested even though I had already
made this requirement previously. I originally had changed the "Primary"
scss variable to $Clear_Cyan: #26BFBF. I have now changed a second "$blue:
#5d95e8". Maybe you missed it the first time because I wanted to keep the
naming consistent with the original Bootstrap file "primary" name. I also
changed, my variable.scss file I name. But the original naming convention
Oliver had told us was okay. I have been busy with my son's baptism all day
today so Im sending this message very late in the even. I hope this will
get to you in time!!

On Sat, Sep 23, 2023 at 12:11 PM Abudu Abidun Sulaiman <
***@***.***> wrote:

> ***@***.**** requested changes on this pull request.
>
> @Pgarza-dev <https://github.com/Pgarza-dev> Here is another chance to
> pass this CA. I just added new comment for you. Failure to implement this
> before the end of today will amount to missing an important requirement of
> the CA and that is an auto failure.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/Pgarza-dev/css-frameworks-ca/pull/1#pullrequestreview-1640869628>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AUBN6PMQVJMILB53SBECVCTX327VZANCNFSM6AAAAAA4HO6QEI>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Mvh/Best Regards

*Pablo Garza *
